.github/workflows/promotion-checker.yml

```diff
@@ -38,7 +38,7 @@ jobs:
 
       - name: Check for PR approval
         run: |
-          APPROVALS=$(gh pr review $PR_NUMBER | grep -c "APPROVED")
+          APPROVALS=$(gh api repos/${{ github.repository}}/pulls/$PR_NUMBER/reviews --jq '[.[] | select(.state == "APPROVED")] | length')
```

Comment:
I changed this to use the github api to check if the PR was approved, and I tested this on the command line as well